Userland/Libraries/LibWeb/Painting/PaintableBox.cpp

```diff
@@ -1010,9 +1010,6 @@ void PaintableBox::set_needs_display(InvalidateDisplayList should_invalidate_dis
 
```

Comment:
Please put the very relevant info from your PR description in the commit message for future reference! :)

Reply:
good idea, updated it :+1:

Reply:
oh, extra lines are still counted towards the commit message length cap :/

Reply:
There is no commit message length cap, only a line length cap. :)